org.bitcoincore.bitcoin-qt.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+
+  <id>org.bitcoincore.bitcoin-qt</id>
+  <name>Bitcoin Core</name>
+  <project_license>MIT</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+
+  <summary>
+    Bitcoin Core connects to the Bitcoin P2P network
+  </summary>
+
+  <url type="homepage">
+    https://bitcoincore.org/
+  </url>
+
```

Comment:
You could add more urls like bugtracker/donate/translations.The specs are here https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html